### PR TITLE
removes cargo techs, makes quartermasters cargo techs

### PR DESCRIPTION
**Pull request by pudl**
-removes cargo tech and related objects
-gives qm 3 slots
-removes clipboard from qm spawn
-removes sunglasses from qm spawn
-gives qm black shoes since brown ones are usually for command
-resprites qm locker to normal cargo locker
-resprites qm pda to normal cargo pda
-changes qm office to cargo monitoring
-gives cargo a mechanical toolbox
-adds 200 metal sheets crate
-adds 200 glass sheets crate
-removes gas mask, firesuit, and internals tank from qm locker
-renames quartermaster's stamp to cargo's stamp
-renames supply shuttle console to quartermaster's console

the rework sheet suggested removing qms and just having cargo techs run the department, that way qm wouldn't be seen as a head anymore. i did that except i prefer the name Quartermaster to Cargo Tech, so i kept it and the jumpsuit.

todo:
-figure out why byond won't let me add a cargo tech jumpsuit to maintenance.

the todo list is trivial, merge this whenever and i won't care.

### CODE REVIEW

**Comment by optimumtact** [comment]
:-1: I do not like the removal of the QM role, it didn't harm anybody and provided a slot for someone to at least learn the ropes of leadership without ruining the station.

**Comment by pudl** [comment]
the hop has nearly no responsibility given the dept he should run the most, cargo, has a mini leader. play hop to learn because its simple to fuck around as him and if cap vanishes its as simple as "no thanks, let ce be cap"

**Comment by Chocobro** [comment]
super against this. a quartermaster is the be-all end-all in a cargo bay. naval, civilian, whatever. having three is completely nonsensical and idiotic. removing the cargo tech job also removes a very basic job that newbies can use to learn basic rp shit and how to work in a team.

**Comment by xxalpha** [comment]
I don't understand this change.

**Comment by Violaceus** [comment]
Please no. There is literally zero reason to remove cargo manager, HoPs are usually busy with other stuff and QM's existence was never a problem.

In fact I think QM should receive proper place among command staff.

**Comment by paprka** [comment]
:-1:

**Comment by RandomMarine** [comment]
A bunch of renames for no reason.
A bunch of resprites for no reason.
Add a couple crates which are just 4x the contents of other crates.
Adds a toolbox.
REMOVES CARGO SHORTS
REMOVES CARGO SHORTS
REMOVES CARGO SHORTS

"Big cargo overhaul"
:-1: :-1: :-1: :-1: :-1: :-1:

**Comment by paprka** [comment]
I don't oppose the idea of cargo being a more tight-knit group but holy shit this basically makes cargo impossible to take by revs since all qms have shades, and if none of them had shades, it would imbalance it the other way.

I don't know who suggested removing the qm/cargo tech dynamic but they are compelte shitters. The HoP oversees supply AS WELL AS service (bar, kitchen, botany) so fuck no

**Comment by as334** [comment]
Also cargo techs are unruly as fuck and need someone to be able to directly lead them, and the HoP isn't able to do that because he's in the office at ALL TIMES! If the HoP has to leave the office in order to go yell at the Quartermasters(That's still really dumb to be multiple of, atleast make it just be multiple cargo techs jesus christ there should only ever be 1 quartermaster or one at all), then you'll have a line of 7+ people by the time you get back, and atleast one of them will probably be dead because of cutting in line. The QM is necessary for maintaining sanity and order in the supply department.

**Comment by Cheridan** [comment]
Yeah, I don't really see the need for this either. I don't think that Cargo is a perfect system, but the dynamics of the QM and cargo techs is something that seems to work.

:-1:

**Comment by as334** [comment]
oh right i forgot the fancy emote :-1:

**Comment by RandomMarine** [comment]
You also overpriced the 200 metal/glass crates.

Ordering 50 sheets costs ten points, yes, but remember that cargo gets 5 points back every crate they send back, so it costs 5 points for the sheets themselves, 5 for the crate. The 200 sheet crates should cost 25 as a result.

**Comment by Fox-McCloud** [comment]
:-1:

If QM being a go between is such a problem then just make him a head of staff and be done with it. I don't personally support this, but it's a lot easier than cutting and thrashing half of cargo.

**Comment by ike709** [comment]
:-1:  just... no...

**Comment by pudl** [comment]
people trying to make the qm a head of staff only make it worse. the reason people view qm as a position of authority is because he serves the job a warden does, except his overseer isn't strong or present enough to show that he ultimately as of the same rank as his coworkers.
the quartermaster/cargo tech synergy is nonexistent, or at least negligible. any sort of direction a qm gives a cargo tech can easily be replicated by the hop directing a new qm, or, perhaps, another qm directing a new qm, since they're all in the same position and there's nothing stopping them from sharing time together, learning. additionally, there is a wiki entry.
the hop has zero obligation to stay in his office and more often than not, he isn't. the hop is officially meant to oversee service and supply. service is a rather self-regulating department since so many people are in it it's hard to pull off crime, and thus supply is left with no overseer except the qm who, ultimately, doesn't really have enough power to actually do anything other than call security.
the hop becoming less of a mini-captain and a head with actual responsibility will make him less a glorified assistant. since supply is a very laid back department, hop can easily serve as a learning role for anyone needing to learn about command.
if you very truly want to play as a role that isn't command and want to boss people around, play warden.

**Comment by dannno** [comment]
for the love of god at least make your commits descriptive of what they are

**Comment by pudl** [comment]
i don't put text in my commits because i always squash them before merge

**Comment by ike709** [comment]
:-1: title should be "Ruining Cargo in one easy PR"

**Comment by Incoming5643** [comment]
Outside of everything else: a position with "master" in its name kind of implies some level of authority over others. Having that be the bottom level job as well as a multiple position job both kind of go against this

**Comment by AlexCrimson** [comment]
Why not just make the QM an official head? Maybe people would take the role more seriously and stop with the Cargonia bullshit.

**Comment by mikecari** [comment]
B-but then the quartermaster will just get lynched by revs all of the days ;_;

**Comment by paprka** [comment]
and now cargo has no contest and will be taken by revs immediately gj pudl

**Comment by pudl** [comment]
working on it

**Comment by Dorsisdwarf** [comment]
When I saw this at first I thought it was just removing the useless crap from the QM's locker, that's how undescriptive the PR is.

Also https://tgstation13.org/wiki/Getting_Your_Pull_Accepted#4._Pull_requests_must_be_atomic

**Comment by Lumipharon** [comment]
This is a pretty fucking retarded idea.

>the quartermaster/cargo tech synergy is nonexistent, or at least negligible. any sort of direction a qm gives a cargo tech can easily be replicated by the hop directing a new qm, or, perhaps, another qm directing a new qm, since they're all in the same position and there's nothing stopping them from sharing time together, learning. additionally, there is a wiki entry.

Literally what? cargotechs tend to be loyal to the QM (when not antags), and even if they're unruly/not listening, removing the role/turning them into a head DOESN'T FUCKING FIX IT. Being a 'head' doesn't magically instil you with charisma any more then being the QM does.
People listen to the warden: They're not a head. The QM is the cargo equivilent of the warden, it's fine as it is.

**Comment by TheVekter** [comment]
So what, we're going back to old Cargo? :-1:

**Comment by Deantwo** [comment]
Wouldn't it have been easier to do it the other way around?
Remove QM and give CargoTech QM access?

Either way, bad idea. :-1:

**Comment by ike709** [comment]
notice how literally everyone is thumbing this down

please take the hint

**Comment by Iamgoofball** [comment]
@ike709 you think this is some kind of democracy?

**Comment by paprka** [comment]
Yeah thumb downs doesn't really matter (and they shouldnt), the pr being bad speaks for itself

**Comment by Chocobro** [comment]
even paprika hates it

**Comment by Jalleo** [comment]
This is not the direction the people who code fortg want I suggest you also consult the people on the forums if you wish to continue.

**Comment by pudl** [comment]
its not a democaracy

**Comment by paprka** [comment]
It's a goodfeaturecracy, and this is terrible

**Comment by AnonymousNow** [comment]
I personally think that this is a terrible, terrible idea. My reasons have already been cited, so I won't run over them again.

**Comment by TheVekter** [comment]
@pudl Care for me to give you a list of why this is an awful idea?

* We changed to having a QM because cargo techs needed someone to keep them from just going CARGONIA every round, and it was a more elegant solution than banning it outright.
* QM is a great trial head position because they get a little bit of authority without being able to fuck a round.
* QM spawns with sunglasses (from what I understand) to make him harder to rev, so rev isn't just >convert cargo and win.
* Constantly removing brown shoes from every fucking job is going to result in no-slips getting meta as fuck ("You're not a head with brown shoes? GIVE ME YOUR SHOES LET'S TEST THEM")

Good things:
* 200 metal and glass crates, thank you based god
* I like them spawning with toolboxes but why not start with a belt instead?

That's about it.

Source: 2 years playing nothing but fucking Cargo Tech before we redid the department

**Comment by Iamgoofball** [comment]
@TheVekter why we don't have chameleon functionality on noslips yet, i don't know

**Comment by Razharas** [comment]
Not good enough

**Comment by hornygranny** [comment]
concept is not valuable

**Comment by Dorsisdwarf** [comment]
generic management phrase

**Comment by Chocobro** [comment]
8======D

**Comment by Miauw62** [comment]
8==================================D~~~~~~((

**Comment by AnonymousNow** [comment]
An additional bonus of the QM's presence - as an extension of Vekter's "trial head" comment, he is the second-highest ranking member of the civil department (unless you count the captain as being part of all departments), good at bureaucracy and handling a desk, in proximity to the HoP's office and, with the warden, one of the two second-in-commands of departments to be directly superior to the remaining staff members (bar the detective for security).

tl;dr - Quartermaster is an ideal candidate to replace the HoP when the HoP gets promoted in the absence of a captain.
